lib/utils.js

```diff
@@ -48,6 +48,7 @@ function getFileExtension({vendor, renderForCaching = false}) {
     case 'deepgram':
     case 'elevenlabs':
     case 'rimlabs':
+    case 'playht':
```

Comment:
It is still not clear when or why we would sometimes cache an mp3 and sometimes a linear file

Reply:
playht support mp3

1/ stream api: freeswitch module download mp3 from playht and convert it to linear.
2/ synthAudio: speech-utils download mp3 and store as mp3

The problem here is because fo stream api play converted linear data from Mp3, then cache the mp3 to redis. next time feature server play the cached mp3 audio that cause different gain compare with linear audio.

Reply:
Getting things working is not enough.  Things also need to be expressed as simply as possible.  Consider the two alternatives:
```
if (not A and not B) {
   do one thing;
else {
   do another;
}
```
versus
```
if (A or B) {
   do another;
}
else {
   do one thing;
}
```
The second is much more understandable.  Can you recast your logic here accordingly?

Reply:
you are right, let me update the code.

I just keep to minimize my changes, but now I modify the legacy code.